### ISSUE TITLE
[UPDATE] Project Statuses 

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#291]
### Is your feature request related to a problem? Please describe.

This full stack React social media app has come to a bit of a stale standstill halt.  It started about 6-7 months ago when I was intending to develop a useable platform if anything, an open source model for one, that can be enjoyable to use.
Admittedly so, I haven't been able to contribute to this project as much as I had hoped to in the beginning.  Some of it due to lack of knowledge the rest due to work obligations taking priority and I am sure that the other maintainers have the same.

My intention was to create a platform to target people over the age of 40-45 if not seniors.  I also had thought about targeting Twitch, Instagram, YouTube, Facebook, X/Twitter, reddit, etc. creators.

Currently we have @manuel12, myself and @pawel975 as core maintainers.  I would like at least 3 more and the possibility of assigning permanent roles in the development of this project whether they be frontend, backend or another role.

I am looking for collaborators, maintainers, contributors to:

- [ ] Create and/or Finish Groups pages and features

- [ ] Create Friends system components and pages and features

- [ ] Create and Fininsh the User Profile and methods

- [ ] Help complete and triage already existing issues

- [ ] UI/UX design with Figma and draw.io

- [ ] Work on writing e2e, unit and integration tests along with @manuel12 

- [ ]  DevOps and Developer Experience  workflows especially newcomers to the project 

- [ ] Docker, .devcontainer and possibly even k8s

- [ ]  Documentation including user, API and Developer docs including internationalization and translation into various language 

- [ ]  work on the Wiki adding docs 

- [ ] Help with maintaining GitHub

- [ ] enhance project by adding Visual Studio Code settings, recommending extensions 

- [ ] A11y Accessibility testing

- [ ] work on creating and maintaining SQLite databases 

- [ ] cross browser and cross  device  testing especially Linux, Mac, Windows, Chrome, Firefox and other devices like laptops  and tablets.

### Describe the solution you would like

Active maintainers, collaborators and contributors.

### Describe the alternatives you have tried

Moving to a GitHub Organization or moving this project to another code hosting platform.

Org mode would allow us to create and develop additional libraries etc on top of RSN.

### Additional context

Let us know how you would like to contribute to this project.

@manuel12 @pawel975

**Comment by Divesh-Kumar-Chordia** [comment]
I would like to contribute 🚀  to your project 📁

**Comment by gbowne1** [comment]
@Divesh-Kumar-Chordia 

Feel free to contribute.  There are plenty of issues to work on.

**Comment by gbowne1** [comment]
@manuel12 where are you at with this project? Let me know

**Comment by manuel12** [comment]
> @manuel12 where are you at with this project? Let me know

Yes, will make an update soon, been  busy with other projects the last few weeks.

**Comment by gbowne1** [comment]
It's been almost a month.  I have not had a lot of time to devote to this project.

**Comment by gbowne1** [comment]
@KitaPDev @balajik

**Comment by gbowne1** [comment]
@Amrani-Farouk-Hossam-Eddine

**Comment by gbowne1** [comment]
@dev-george-nikolaidis
